src/components/home/Reservations/index.js

```diff
@@ -52,54 +51,58 @@ export class Reservations extends Component {
   }
 
   renderReservations = () => {
-    if (this.state.loading) return <Text>'loading...'</Text>
-    const reservations = this.state.reservations.map((session, index) => {
+    if (this.state.loading) return <MyText>'loading...'</MyText>
```

Comment:
return this.state.loading && <MyText>'loading...'</MyText> 

ES6 vieux haha

Reply:
&& c'est pas de l'ES6, et j'ai besoin du return pour sortir de la fonction